hbase-server/src/main/java/org/apache/hadoop/hbase/regionserver/HRegion.java

```diff
@@ -7177,8 +7177,8 @@ HRegion createDaughterRegionFromSplits(final HRegionInfo hri) throws IOException
     fs.commitDaughterRegion(hri);
 
     // Create the daughter HRegion instance
-    HRegion r = HRegion.newHRegion(this.fs.getTableDir(), this.getWAL(), fs.getFileSystem(),
-        this.getBaseConf(), hri, this.getTableDesc(), rsServices);
+    HRegion r = HRegion.newHRegion(this.fs.getTableDir(), rsServices.getWAL(hri),
```

Comment:
> rsServices.getWAL(hri)
Based on WALProvider impl, it will create a new wal according to strategy or inherit parent by default mode.